src/ocm/components/AddHosts/HostsClusterDetailTab.tsx

```diff
@@ -85,6 +87,19 @@ const HostsClusterDetailTabContent: React.FC<HostsClusterDetailTabProps> = ({
     if (isVisible && day2Cluster === undefined && cluster && openshiftClusterId && pullSecret) {
       // ensure exclusive run
       setDay2Cluster(null);
+
+      const openshiftVersion = normalizeClusterVersion(cluster.openshift_version);
+      if (!openshiftVersion) {
+        setError(
+          <>
+            Unsupported OpenShift cluster version: ${cluster.openshift_version}.
+            <br />
+            Check your connection and <TryAgain />.
+          </>,
+        );
```

Comment:
```suggestion
        setError(
          <>
            Unsupported OpenShift cluster version: ${cluster.openshift_version}.
            <br />
            Check your connection and <TryAgain />.
          </>,
        );
```
- Does the connection have anything to do with the fact the cluster's OCP version is unsupported?
- Trying again would help somehow?

Reply:
Let's merge it as is, since the benefit of having this is better than nothing. @jgyselov please keep working on enhancing this specific text with the UXD team.

Reply:
I was pretty much just reverting to https://github.com/openshift-assisted/assisted-ui-lib/pull/1365.

I can talk to UXD about this text or perhaps a better option would be to test the openshift version before we display the Add Hosts tab? Anyway, I've created a [Jira issue](https://issues.redhat.com/browse/MGMT-11804) for it, we can continue the conversation there.

@jkilzi